### ISSUE TITLE
Section 508 tags in rule-descriptions.md either missing or inconsistent

### ISSUE THREAD

**Issue by iamrafan** [dequelabs/axe-core#578]
The following best practice rules in rule-descriptions are either missing Section 508 tags or the tags associated with them are inconsistent with those in the respective rule pages

Rule ID | Rule-descriptions list | Individual rule page
-- | -- | --
accesskeys |   | 1194.21 (a)
aria-hidden-body |   | 1194.22 (d)
checkboxgroup |   | 1194.21 (l),   1194.22 (n)
color-contrast |   | 1194.21 (j)
frame-title-unique |   | 1194.22 (i)
image-redundant-alt |   | 1194.22 (e)
label-title-only |   | 1194.21 (l),   1194.22 (n)
link-in-text-block |   | 1194.21 (j)
meta-refresh |   | 1194.22 (p)
radiogroup |   | 1194.21 (l),   1194.22 (n)
scope-attr-valid |   | 1194.22 (g),   1194.22 (h)
skip-link |   | 1194.22 (o)
table-fake-caption | section508.22.g | 1194.22 (g),   1194.22 (h)
td-has-header | section508.22.g | 1194.22 (g),   1194.22 (h)
td-headers-attr | section508.22.g | 1194.22 (g),   1194.22 (h)
th-has-data-cells | section508.22.g | 1194.22 (g),   1194.22 (h)

**Comment by marcysutton** [comment]
I don't think this is useful to fix as Section 508 has aligned with WCAG 2.0. @WilcoFiers any input on this? It also doesn't seem that relevant for best practice rules, in any case.

**Comment by marcysutton** [comment]
I've made some edits to rule tags, and will open a PR in a moment. But there are a few on this list that are best practice rules, so they won't have WCAG or Section 508 tags. Any discrepancies on help pages should be resolved in https://github.com/dequelabs/axe-core/issues/575.

Best practice rules
- checkboxgroup
- frame-title-unique
- image-redundant-alt
- label-title-only
- radiogroup
- scope-attr-valid
- skip-link

**Comment by marcysutton** [comment]
This should be resolved on Deque U.
